web/frontend/src/components/slam.vue

```diff
@@ -392,6 +422,14 @@ onUnmounted(() => {
       slot="title"
       crumbs="slam"
     />
+    <v-button
+      slot="header"
+      variant="danger"
+      icon="stop-circle"
+      :disabled="!moveClicked ? 'true' : 'false'"
```

Comment:
[nit] 
```suggestion
      :disabled="moveClicked ? 'false' : 'true'"
```

Or if you prefer having true first, then replicate this on line 531